### PR TITLE
[BUGFIX] Freeze boto3

### PR DESCRIPTION
**Pull request by barry-jin**
## Description ##
Fix issue #1369 

### Essentials ###
- [x] PR's title starts with a category (e.g. [BUGFIX], [MODEL], [TUTORIAL], [FEATURE], [DOC], etc)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage
- [x] Code is well-documented

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

cc @dmlc/gluon-nlp-team

### CODE REVIEW

**Review by szha — APPROVED** [review]

File: setup.py [hunk 1]
```diff
@@ -76,7 +76,7 @@ def find_version(*file_paths):
     install_requires=requirements,
     extras_require={
         'extras': [
-            'boto3',
+            'boto3==1.14.32',
```
Comment (leezu) [inline setup.py:79]: How about specifying an acceptable arrange, as this affects all users installing with `[extra]`

Comment (barry-jin) [reply setup.py:79]: `awscli` and `boto3` all require `botocore` this dependency. `botocore 1.18.0` has removed doc files and migrated to `awscli >= 1.18.140`. Since we installed [`awscli`](https://github.com/dmlc/gluon-nlp/blob/56de303f2283937cca5c0e67703713d624b6fd05/tools/batch/docker/Dockerfile.gpu#L29)during building the docker file and installed [`boto3`](https://github.com/dmlc/gluon-nlp/blob/56de303f2283937cca5c0e67703713d624b6fd05/tools/batch/docker/gluon_nlp_job.sh#L23) when boot up an AWS batch job. So, to solve docevents missing issue, we can either keep `boto3` and `awscli` all in old version or update `awscli` by updating the docker and take `boto3` in newest version.

**Comment by barry-jin** [comment]
> The documentation website for preview: http://gluon-nlp-dev.s3-accelerate.amazonaws.com/PR#1370/freeze-boto3/index.html

The documentation website for preview: http://gluon-nlp-dev.s3-accelerate.amazonaws.com/PR%231370/freeze-boto3/index.html

**Comment by szha** [comment]
the url looks weird. if there's trouble in handling `#` character, we can do without it.
